### PR TITLE
No longer require Haste

### PR DESCRIPTION
**Pull request by qzminski**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,10 +26,8 @@
         "source": "https://github.com/terminal42/contao-ajaxform"
     },
     "require": {
-        "php": "^5.3.2 || ^7.0 || ^8.0",
-        "contao/core-bundle": "~3.5 || ~4.1",
-        "contao-community-alliance/composer-plugin": "^2.4.1 || ~3.0",
-        "codefog/contao-haste": "4.*"
+        "php": "^7.4 || ^8.0",
+        "contao/core-bundle": "^4.13 || ^5.0"
```
Comment (Toflar) [inline composer.json:30]: Given that 5.1 will likely get this feature merged, I think we should probably only allow `5.*`? Wdyt?

Comment (qzminski) [reply composer.json:30]: If it makes its way to 5.1, then we should update the composer.json and archive the repository, yeah. Not sure if we should do it now, though.

Comment (Toflar) [reply composer.json:30]: I would like to have this adjusted to `5.0.*`, please. If the PR does not make it to 5.1, we can still release a new hotfix version here and allow 5.1. However, if we merge this as is, we can never go back to not allowing > 5.0 😉

Comment (qzminski) [reply composer.json]: Done!

**Review by Toflar — APPROVED** [review]
Thanks! Because we increased the minimum requirements, I think this deserves a new minor. So release as 1.2.0 :)

File: composer.json [hunk 2]
```diff
@@ -26,10 +26,8 @@
         "source": "https://github.com/terminal42/contao-ajaxform"
     },
     "require": {
-        "php": "^5.3.2 || ^7.0 || ^8.0",
-        "contao/core-bundle": "~3.5 || ~4.1",
-        "contao-community-alliance/composer-plugin": "^2.4.1 || ~3.0",
-        "codefog/contao-haste": "4.*"
+        "php": "^7.4 || ^8.0",
+        "contao/core-bundle": "^4.13 || 5.0.*"
```
Comment (aschempp) [inline composer.json:30]:
```suggestion
        "contao/core-bundle": "^4.9 || 5.0.*"
```

**Review by aschempp — CHANGES_REQUESTED** [review]

**Comment by BoozieBadd** [comment]
👋📱

File: AjaxForm.php [hunk 3]
```diff
@@ -105,9 +104,8 @@ protected function jumpToOrReload($intId, $strParams = null, $strForceLang = nul
      */
     private static function sendResponse($content)
     {
-        $insertTags = new \Contao\InsertTags();
-        $content = $insertTags->replace($content, false);
-        $objResponse = new HtmlResponse($content);
-        $objResponse->send();
+        $content = \Contao\System::getContainer()->get('contao.insert_tag.parser')->replaceInline($content);
```
Comment (aschempp) [inline AjaxForm.php:107]: the `contao.insert_tag.parser` service does not exist in Contao 4.9 (which is supported according to `composer.json`)!

Comment (qzminski) [reply AjaxForm.php:107]: This pull request has been already merged, and this problem already fixed afterward: https://github.com/terminal42/contao-ajaxform/commit/17cd66cb2acef3b7ff3e8adf0dcc9b367bdc4abc

**Pull request merged**